sway-core/src/language/ty/expression/expression_variant.rs

```diff
@@ -880,7 +882,14 @@ impl ReplaceSelfType for TyExpressionVariant {
 }
 
 impl ReplaceDecls for TyExpressionVariant {
-    fn replace_decls_inner(&mut self, decl_mapping: &DeclMapping, engines: &Engines) {
+    fn replace_decls_inner(
+        &mut self,
+        decl_mapping: &DeclMapping,
+        handler: &Handler,
+        ctx: &TypeCheckContext,
+    ) -> Result<(), ErrorEmitted> {
+        let mut error_emitted = None;
```

Comment:
Same `handler.scope` suggestion as above

Reply:
Thanks for the suggestion, most recent version now uses scopes.